src/tests/e2e/login.test.e2e.js

```diff
@@ -18,4 +27,43 @@ describe('Login and onboarding', () => {
     },
     timeout
   )
+
+  it(
+    'Login as a new user',
+    async () => {
+      // close non in
+      await page.close()
+      // first login to disconnect page to delete cloud wallet
+      let context = await browser.createIncognitoBrowserContext()
+      // Create a new page inside context.
+      page = await context.newPage()
+
+      await page.goto(`${process.env.E2E_TEST_URL}/disconnect`)
+      const loginPage = new LoginPage()
+      await loginPage.login(
```

Comment:
aren't we at disconnect page, how do we login?

Reply:
the '/disconnct' requires login first